app/models/user.rb

```diff
@@ -3,13 +3,30 @@ class User < ActiveRecord::Base
   # :confirmable, :lockable, :timeoutable and :omniauthable
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :trackable, :validatable,
-         :omniauthable, omniauth_providers: [:twitter]
+         :omniauthable, omniauth_providers: [:twitter],
+         :authentication_keys => [:login]
```

Comment:
__[rubocop]__
* Style/AlignHash: Align the elements of a hash literal if they span more than one line.
* Style/HashSyntax: Use the new Ruby 1.9 hash syntax. [:ref](https://github.com/bbatsov/ruby-style-guide#hash-literals)